packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -52,11 +54,13 @@ type TParsedHttpAdapterFlags = Record<
 // is no need to be concerned about security.
 const ldClientSideIdProduction = '5979d95f6040390cd07b5e01';
 
-function getUserCustomFieldsForLaunchDarklyAdapter(
+function getUserContextForLaunchDarklyAdapter(
   user?: Props['user'],
   projectKey?: string
 ): TLaunchDarklyUserCustomFields {
   return {
+    kind: 'user',
+    key: user?.id,
```

Comment:
Ja, good idea. With the new setup in flopflip this would actually work. With v12 it wouldn't have as the `key` would have been generated if not provided.